src/components/projects/CreateProjectForm.vue

```diff
@@ -1230,7 +1397,8 @@ export default {
       'available_sites_simulated',
       'filter_wheel_options',
       'default_filter_selection',
-      'selected_mount_config'
+      'selected_mount_config',
+      'get_camera_config'
```

Comment:
can we call this just `camera_config`? just to match the other naming conventions for the getters